diff --git a/node_modules/strapi-plugin-ckeditor5/admin/src/config/ckeditor.js b/node_modules/strapi-plugin-ckeditor5/admin/src/config/ckeditor.js
index ca99041..aa9156d 100644
--- a/node_modules/strapi-plugin-ckeditor5/admin/src/config/ckeditor.js
+++ b/node_modules/strapi-plugin-ckeditor5/admin/src/config/ckeditor.js
@@ -164,8 +164,11 @@ module.exports = {
     // ],
   },
   htmlEmbed: {
-    // showPreviews: true,
+    showPreviews: true,
   },
+  mediaEmbed:{
+    previewsInData: true,
+      },
   fontFamily: {
     // options: [
     //   "default",
@@ -181,6 +184,7 @@ module.exports = {
     //   "Lato, Inter, sans-serif",
     // ],
   },
+
   link: {
     // defaultProtocol: "http://",
     // decorators: [
